### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by OybekKayumov**
### In this branch:
---

- Removed all boilerplate from CRA
- Created components folder
- Inside components created Calculator.js
- In Calculator.js created a React component that match given design
- App works without problems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly format your README using this [Microverse template](https://github.com/microverseinc/readme-template). Note that all the sections are required except for the Tests, Deployment, and Live demo sections if you do not have them.

- [ ] Kindly add more detailed instructions on how to set up and run your app locally, ie cloning the repo, installing packages, and starting the app.

- [ ] You are yet to delete all boilerplate files. Kindly delete the `public/robots.txt` file.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="box-container">
+        <div className="result-screen">0</div>
```
Comment (TedLivist) [inline src/components/Calculator.js:13]: - [ ] Kindly ensure your design matches the given template.

  Your Implementation | Design template
  :---:|:---:
  ozb-1 | ozb-2

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @OybekKayumov,

Good job so far!
✔️ Use of class-based components
✔️ Calculator component is in appropriate directory
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (OybekKayumov) [reply README.md]: readme updated
public/robots.txt deleted

File: src/components/Calculator.js (hunk 2 above)
Comment (OybekKayumov) [reply src/components/Calculator.js:13]: updated

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @OybekKayumov,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**